.github/workflows/security-scorecard.yml

```
@@ -3,6 +3,8 @@ name: Security Scorecard
 on:
   push:
     branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)